net/daq/Portfile

```diff
@@ -17,12 +17,22 @@ depends_lib         port:pcre \
                     port:libdnet \
                     port:libpcap
 
-checksums           rmd160  df1f292e4f91fea5e66db54581d1952e7be3771d \
-                    sha256  d41da5f7793e66044e6927dd868c0525e7ee4ec1a3515bf74ef9a30cd9273af0 \
-                    size    518013
-# There was a stealth update of the distfile - hence the next line is required
-dist_subdir         ${name}/${version}_1
-patchfiles          patch-sfbpf-Makefile.in.diff
+checksums           rmd160  c915d34c0ba6c7deb469d6969bc1e077a27cf576 \
+                    sha256  bdc4e5a24d1ea492c39ee213a63c55466a2e8114b6a9abed609927ae13a7705e \
+                    size    520287
+
+pre-configure {
+    delete ${worksrcpath}/configure
```

Comment:
Why does the configure script need to be deleted? Doesn't running autoreconf automatically overwrite it?

Reply:
Well this is a weird one, on my system autoreconf can’t delete it and spews out permission denied. Maybe you can try on yours and see if it also happens?

Reply:
Maybe due to another packaging error: none of the files have the write permission bit set.